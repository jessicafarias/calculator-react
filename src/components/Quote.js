const Quote = () => (
  <div className="bg-image2">
    <div className="div-center">
      <h1>Quote</h1>
      <p>
        {' '}
        Mathematics is not about numbers, equations,
        computations, or algorithms: it is about understanding.
        â€“William Paul Thurston
      </p>
    </div>
  </div>
);

export default Quote;
