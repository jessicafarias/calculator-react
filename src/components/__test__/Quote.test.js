import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer'; // ES6
import Quote from '../Quote';

describe('Display Quote page', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<BrowserRouter><Quote /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
