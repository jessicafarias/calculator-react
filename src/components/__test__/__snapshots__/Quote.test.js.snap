// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Quote page renders correctly 1`] = `
<div
  className="bg-image2"
>
  <div
    className="div-center"
  >
    <h1>
      Quote
    </h1>
    <p>
       
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
    </p>
  </div>
</div>
`;
